packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -364,16 +520,176 @@ const Headers = ({ titles, className }: { titles: string[], className: string })
   );
 };
 
+const Checkpoint: React.FC<{ label: string; status: 'done' | 'current' | 'not_started' }> = ({ label, status }) => {
+  let backgroundColor;
+  let borderColor;
+  let size;
+  let labelColor;
+
+  switch (status) {
```

Comment:
This styling handling is done outside of our usual system, likely isn't dark-mode safe, etc.

Reply:
However looks like this is unused, if so, remove